client.go

```diff
@@ -120,6 +127,8 @@ type SessionInfo struct {
 type NativeClient struct {
 	HostDetails         []HostDetail // list of Hosts
 	ClientVersion       string       // ClientVersion is the version string to send to the server when identifying
+	persistentConn      *ssh.Client  // cache client
+	persistentConnMux   sync.Mutex
```

Comment:
Regarding Jim's comment, it would be most accurate to call this the "persistentConnectedClient", because that's what we did - took an ssh.Client and did a dial so it's connected already, and we can spawn sessions from this connected client. But that's a mouthful. Maybe just "connectedClient"?